### ISSUE TITLE
Clearpath Realsense integration has incorrect parameter fields in configuration file

### ISSUE THREAD

**Issue by SteveMacenski** [clearpathrobotics/clearpath_robot#66]
The robot was shipped with the following fields set https://github.com/clearpathrobotics/clearpath_robot/blob/main/clearpath_sensors/config/intel_realsense.yaml#L12-L14 as `640,240,30`, yet when I modified them nothing happened.

It turned out this is fundamentally the incorrect parameter name. It is `depth_module.depth_profile` and `rgb_camera.color_profile`.

**Comment by SteveMacenski** [comment]
Not bug, but just for clarity for those that come later, since this took me an hour to figure out myself:

If you enable the decimation filter, you can modify the kernal with the `filter_magnitude` field to get it down to a reasonable size for navigation usage

```
    decimation_filter.enable: True
    decimation_filter.filter_magnitude: 6
```

Additionally, remove all `stream_filter` and `stream_index_filter`'s to `0` to get rid of the color which adds to the transport size, if not using the color directly for applications.

**Comment by luis-camero** [comment]
Fixed in #67.
